PROCESSORS.md

```diff
@@ -704,6 +704,10 @@ In the list below, the names of required properties appear in bold. Any other pr
 | Name | Description |
 | - | - |
 |success|All files are routed to success|
```

Comment:
I think this could be clarified, eg. `All incoming FlowFiles are routed, unchanged, to succes`

Reply:
This is not true, I updated it in ddb119bc9a787d6d5c8600dfd2c7ed22e72da1ed to be on par with the NiFi description.